src/components/TokenBalance.tsx

```diff
@@ -203,7 +209,13 @@ export function AssetsTokenBalance({ showInfo }: { showInfo: boolean }) {
   )
 }
 
-export function HomeTokenBalance() {
+export function HomeTokenBalance({
+  buttonOnPress,
+  hideBalance = false,
+}: {
+  buttonOnPress?: () => void
```

Comment:
nit: rename to be more descriptive. maybe `eyeIconOnPress`?
Also should the props be required instead of optional? Seems like the eye icon is rendered always and there needs to be an onPress passed.